cassandra-oss/Dockerfile.ubi7

```diff
@@ -0,0 +1,183 @@
+# Copyright DataStax, Inc, 2020
+#   Please review the included LICENSE file for more information.
+#
+ARG BASETAG=7.8
+FROM registry.access.redhat.com/ubi7/ubi-minimal:${BASETAG} AS builder
+
+ARG URL_PREFIX=https://dlcdn.apache.org/cassandra
+ARG CASSANDRA_VERSION=trunk
+ARG METRICS_COLLECTOR_VERSION=0.3.4
+ARG TINI_VERSION=0.19.0
+
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
+
+# Update base layer
+RUN microdnf update && rm -rf /var/cache/yum
+
+# Install packages needed during install process
+RUN microdnf install shadow-utils tar gzip unzip && microdnf clean all
+
+# COPY in files if they are available locally
+COPY * /
+WORKDIR /
+
+# Download and extract Metrics Collector
+ENV MCAC_PATH /opt/metrics-collector
+RUN mkdir ${MCAC_PATH} && \
+    if test ! -e datastax-mcac-agent-0.1.9.tar.gz; then curl -L -O "https://github.com/datastax/metric-collector-for-apache-cassandra/releases/download/v${METRICS_COLLECTOR_VERSION}/datastax-mcac-agent-${METRICS_COLLECTOR_VERSION}.tar.gz"; fi && \
+    tar --directory ${MCAC_PATH} --strip-components 1 --gzip --extract --file datastax-mcac-agent-${METRICS_COLLECTOR_VERSION}.tar.gz
```

Comment:
As it turns out, the latest MCAC (v0.3.4) doesn't work with Cassandra trunk. So either I can pull this out, or we have to update MCAC to work with trunk.